CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ Embedded Graphics is a `no_std` library for adding graphics features to display
 
 ### Changed
 
+- **(breaking)** - [#638](https://github.com/embedded-graphics/embedded-graphics/pull/638) Bump all crate Minimum Supported Rust Versions (MSRV) to 1.56.1, aka edition 2021.
```

Comment:
While this PR doesn't set any hard rules on MSRV, it changes it in the docs and CI so I thought it appropriate to add the changelog entry for it instead of a future one with more significant changes.

Reply:
Agreed.